redis/internal/redisprom/metrics.go

```diff
@@ -0,0 +1,45 @@
+package redisprom
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/reddit/baseplate.go/prometheusbp"
+)
+
+// Baseplate.spec Prometheus metrics
+// https://github.snooguts.net/reddit/baseplate.spec/blob/master/component-apis/prom-metrics.md#redis
```

Comment:
this is an internal link which we probably should not put into public code.

Reply:
gone